packages/widgetbook/lib/src/fields/date_time_field.dart

```diff
@@ -0,0 +1,136 @@
+import 'package:flutter/material.dart';
+
+import 'field.dart';
+import 'field_codec.dart';
+import 'field_type.dart';
+
+extension DateTimeExtension on DateTime {
+  /// Converts the [DateTime] to a string object supported by the field
+  String toSimpleFormat() {
+    String pad(int value) {
+      return value.toString().padLeft(2, '0');
+    }
+
+    return '$year-${pad(month)}-${pad(day)} ${pad(hour)}:${pad(minute)}';
+  }
+}
+
+extension DateStringExtension on String {
+  /// Converts the string gotten from our [DateTimeExtension.toSimpleFormat]
+  /// to a [DateTime] object manually since [DateTime.parse] doesn't work
+  DateTime fromSimpleFormat() {
+    final parts = split(RegExp(r'[- :]+'));
+    return DateTime(
+      int.parse(parts[0]),
+      int.parse(parts[1]),
+      int.parse(parts[2]),
+      int.parse(parts[3]),
+      int.parse(parts[4]),
+    );
+  }
+}
```

Comment:
`DateTime.parse("2023-10-09 05:00")` should work, are you sure we need this and it's not a side-effect of another error?

If that's still needed I would call the method `toDateTime` instead of `fromSimpleFormat`, what do you think of that new name?

Reply:
Yeah, it actually works, what I had in mind was when strings like `2023-1-9 05:00` fail because each block doesn't have a minimum of two digits.
I removed the `fromSimpleFormat` if you prefer the default parser instead

Reply:
I would prefer if we have went with the **simpler approach** as long as it **works properly**. So I would leave that decision for you, I trust you judgement here.